libs/rtemodel/include/CprjFile.h

```diff
@@ -319,6 +320,7 @@ class CprjFile : public RtePackage
   */
   RteItem* GetProjectInfo() const;
 
+  const std::optional<std::string> GetRteFolder() const;
```

Comment:
Which advantage optional<> brings here to compare to string?

Reply:
Maybe it is mostly personal preference, but for me the benefit of using optional is that it reduces the mental load on the programmer since I now don't have to remember when empty_string is considered either as OK, or usable value.
I can trust the created optional to "remember" this for me.

At the time when we create the optional, that is when we should take the decision if empty_string is an allowed value or not. Each user of the value should not have to remember this, but again, may be a personal preference thing.

Reply:
Agreed, this might have sense